mock-api/getApiResult.js

```diff
@@ -0,0 +1,61 @@
+import fs from 'fs';
+import path from 'path';
+import update from 'react-addons-update';
+import {
+  getChartTypeDefaultOpts,
+  getChartTypeObject,
+} from '../app/utils/chartTypeUtils';
+import { parseRawData, transformParsedData } from '../app/utils/rawDataHelpers';
+import Baby from 'babyparse';
```

Comment:
The frontend uses Papaparse, which I chose for its error handling. Babyparse is just to keep the backend of the fake API more-or-less on par with that.